core/whistle_apps-1.0.0/src/whapps_controller.erl

```diff
@@ -59,13 +59,12 @@ start_app(App) ->
 stop_app(App) when not is_atom(App) ->
     stop_app(wh_util:to_atom(App));
 stop_app(App) ->
-    _ = application:stop(App),
-    lager:info("stopped kazoo application ~s", [App]).
+    case application:stop(App) of
+      ok -> lager:info("stopped kazoo application ~s", [App]), ok;
+      Err -> Err
+    end.
 
--spec restart_app(atom() | nonempty_string() | ne_binary()) ->
-                         {'ok', pid() | 'undefined'} |
-                         {'ok', pid() | 'undefined', term()} |
-                         {'error', term()}.
+-spec restart_app(atom() | nonempty_string() | ne_binary()) -> 'ok' | {'error', term()}.
```

Comment:
this spec seems wrong, it doesn't consider the values returned by start_app which is what is returned.

shouldn't this be
-spec restart_app(atom() | nonempty_string() | ne_binary()) -> 'ok' | 'error' | 'exists'.